Formula/prometheus.rb

```diff
@@ -22,12 +22,20 @@ def install
     libexec.install %w[consoles console_libraries]
   end
 
+  def post_install
+    system "echo '--config.file #{etc}/prometheus.yml  \
+      --web.listen-address=127.0.0.1:9090 \
+      --storage.tsdb.path #{var}/prometheus' \
+      > #{etc}/prometheus.args"
```

Comment:
This might be a better way
```ruby
    (etc/"prometheus.args").write <<~EOS
      --config.file #{etc}/prometheus.yml
      --web.listen-address=127.0.0.1:9090
      --storage.tsdb.path #{var}/prometheus
    EOS
```

Reply:
good idea. Changed.